### ISSUE TITLE
Playing selected audio only works once

### ISSUE THREAD

**Issue by thomaskisler** [IPS-LMU/EMU-webApp#140]
Calling the WebApplication from WebMAUS in Chrome, selecting a part of the audio to play in the Spectogram and pressing "c". The audio plays once (or until stopped by another "c") and then doesn't play anymore (even though the increasing gray bar implies otherwise), until a new selection is made.

**Comment by thomaskisler** [comment]
Okay, after some more testing it does not only affect "c", but also playing back the audio in other ways. Google-Chrome version 46.02490.71 (64-bit) and 47.0.2516.0 dev (64-bit) (47.xx is currently unstable) produce the following error when the sound dissappears:

[23:23:1015/153410:ERROR:render_media_log.cc(20)] MediaEvent: PIPELINE_ERROR demuxer: could not open
[23:23:1015/153410:ERROR:render_media_log.cc(20)] MediaEvent: PIPELINE_ERROR pipeline: read error

**Comment by raphywink** [comment]
I just put together this page: http://www.phonetik.uni-muenchen.de/~raphael/chrome46audioBug/

After the initial load both players work fine. If you then reload the page several times the `src="data:audio/wav;base64,UklGR` player stops working... leading to the same errors mentioned above...

**Comment by raphywink** [comment]
Same errors occur on Windows 7 professional and OSX 10.11! On OSX 10.11 if you start chrome in the command line you even get the same:

```
[14081:1295:1019/164916:ERROR:render_media_log.cc(20)] MediaEvent: PIPELINE_ERROR demuxer: could not open
[14081:1295:1019/164916:ERROR:render_media_log.cc(20)] MediaEvent: PIPELINE_ERROR pipeline: read error
```

errors.

**Comment by raphywink** [comment]
As this is a huge deal for us I just reported the bug here: https://code.google.com/p/chromium/issues/detail?id=544988&thanks=544988&ts=1445268485 ... let's see what they have to say!

**Comment by MJochim** [comment]
The same thing happens on Arch Linux with PulseAudio.

**Comment by raphywink** [comment]
Closing this issue as it is fixed as of version 0.0.28. The src=bas64string problem of the audio tag is still broken! Follow the chrome bug report for this...
